src/components/Quote.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import './quote.css';
+
+const Quote = () => {
+  const [quote, setQuote] = useState([
+    {
+      quote: '',
+      author: '',
+    },
+  ]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const url = 'https://api.api-ninjas.com/vi/quotes';
+
+  useEffect(() => {
+    const getQuote = async () => {
+      setIsLoading(true);
+      try {
+        const fetchQuote = await fetch(url, {
+          method: 'GET',
+          header: {
+            'Content-Type': 'application/json',
+            'X-Api-Key': 'tgGQLF0ayPX9ZzYdnazDEA==ZUlyDiE4HJM8YHuy',
+          },
+        });
+        const fetchedQuote = fetchQuote.json();
+        setQuote(fetchedQuote);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    getQuote();
+  }, [setQuote, setIsLoading]);
+
+  if (hasError) {
+    return (
+      <div className="Quote-container Quote-filler">
+        <span className="error">Error getting the quotes!!!</span>
+      </div>
+    );
+  }
+  if (isLoading) {
+    return (
+      <div className="Quote-container Quote-filler">
+        <span>Loading the quotes...</span>
```

Comment:
- [ ] Good job so far 👏 👏 However while I was testing the app, I found out that whenever I refresh the page, I saw that there is an `Error getting the quotes!!!` That means that there is a problem in the way you are making fetch requests and probably it is because of the  `unauthorized request` maybe the API key is not valid or things like that. Therefore I would suggest that you check the fetch request again and make sure it is working because currently I can't get any quote 😃